### PR TITLE
Set ScriptHref to null if we can't find a script file for the function

### PR DESCRIPTION
**Pull request by ahmelsayed**

### CODE REVIEW

File: Kudu.Core/Functions/FunctionManager.cs [hunk 1]
```diff
@@ -296,17 +296,13 @@ private string GetFunctionScriptPath(string functionName, JObject functionInfo)
...
-                    // TODO: should this be an error?
-                    return functionPath;
-                }
                 return functionPrimary;
             }
         }
 
         private Uri FilePathToVfsUri(string filePath, bool isDirectory = false)
         {
+            if (string.IsNullOrEmpty(filePath)) return null;
```
Comment (suwatch) [inline Kudu.Core/Functions/FunctionManager.cs]: nit using {  and } for readability.

Comment (suwatch) [inline Kudu.Core/Functions/FunctionManager.cs]: is there a reason not not fix the caller instead (never passing null)?  callee should throw if null or empty etc.   easier to find bug that way.

**Comment by davidebbo** [comment]
@ahmelsayed what's the status of this PR?

**Comment by davidebbo** [comment]
@ahmelsayed Can you update status on this one?

**Comment by ahmelsayed** [comment]
obsolete by #2008

**Pull request closed without merging**